front-end/src/app/reports/transactions/transaction-list/transaction-receipts/transaction-receipts.component.html

```diff
@@ -80,7 +80,12 @@ <h5 class="gandhi-bold size-24">Receipts</h5>
           <div *ngIf="item?.itemized === false"><strong>Unitemized</strong></div>
         </td>
         <td>
-          {{ item.name }}
+          {{
```

Comment:
`name` was not being serialized correctly in the api.  it's fixed in the e2e ticket and can be used again here (since we're sorting on the name value)